crates/subspace-runtime-primitives/src/lib.rs

```diff
@@ -147,3 +147,11 @@ pub mod opaque {
     /// Opaque block identifier type.
     pub type BlockId = generic::BlockId<Block>;
 }
+
+/// A trait for finding the address for a block rewards based on the `PreRuntime` digests contained within it.
+pub trait FindBlockRewardsAddress<Address> {
+    /// Find the address for a block rewards based on the pre-runtime digests.
+    fn find_block_rewards_address<'a, I>(digests: I) -> Option<Address>
```

Comment:
`rewards_address` here and `reward_address` in other places seemingly lack some consistency :D

Reply:
Block reward address is better I think, we'll have other rewards (we already have them, but everything goes to block author, so doesn't matter too much).

Reply:
The method is still `find_block_rewards_address`, not `find_block_reward_address`

Reply:
`FindBlockRewardsAddress` should be `FindBlockRewardAddress` as well.